src/wallet/scriptpubkeyman.cpp

```diff
@@ -1721,37 +1721,157 @@ std::unordered_set<CScript, SaltedSipHasher> LegacyDataSPKM::GetScriptPubKeys()
     // For all keys, if they have segwit scripts, those scripts will end up in mapScripts
     for (const auto& script_pair : mapScripts) {
         const CScript& script = script_pair.second;
-        if (IsMine(script) == ISMINE_SPENDABLE) {
```

Comment:
In 905e22b469a1e09df9ff0e98bf989a55642f301e "wallet: Remove IsMine from migration": Maybe it's worth updating the documentation?

```
// For every script in mapScript, only the ISMINE_SPENDABLE ones are being tracked.
```

Reply:
Updated the comment.